src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpPrincipal.java

```diff
@@ -75,13 +72,17 @@ public String getName() {
 
     /**
      * returns the username this object was created with.
```

Comment:
Capitalize and add a period.
Please check all methods.

Reply:
Will include change in https://bugs.openjdk.java.net/browse/JDK-8252822